package.json

```diff
@@ -24,12 +24,7 @@
     "LICENSE.md",
     "README.md"
   ],
-  "licenses": [
-    {
-      "type": "The MIT License",
-      "url": "http://www.opensource.org/licenses/mit-license.php"
-    }
-  ],
+  "license": "MIT",
```

Comment:
It continued to give a warning saying that the license was invalid when testing. Documentation is here showing deprecation https://docs.npmjs.com/files/package.json#license